src/main.py

```diff
@@ -10,20 +10,24 @@
 from commonbot.debug import Debug
 from commonbot.timekeep import Timekeeper
 
+# Needs to happen before other imports that cause db to be queried
```

Comment:
Out of curiosity, why?

Reply:
This fixes bouncer failing to start if you don't have a database file.

If you didn't have a db file, then the initialization behavior was:
 - `import commands`
 - `commands.py` creates a blocked users instance
 - `BlockedUsers`'s init function calls `db.get_blocklist()`
 - This fails because the db wasn't initialized yet so there are no tables

Now, the order is:
- `import db` then `db.initialize()`
- everything else

To prevent this issue/similar ones in the future.